_sass/_main.scss

```diff
@@ -388,10 +389,40 @@ a {
   }
 }
 
-
 // Individual school pages
 //============================================
 
 .school-section {
   border-bottom: $regular-border-size solid $base-border-color;
+  @include outer-container();
+
+  h1 {
+    padding-top: 0;
+  }
+
+  h2 {
+    margin-top: 0;
+    margin-bottom: 0;
+    padding-top: 0;
+    font-size: font-size($h4);
+    font-weight: normal;
```

Comment:
All properties should be in ABC order inside the general suggested order -- which is, from the styleguide:
1. variables
2. @extend directives
3. @include directives
4. properties
5. nested tags
6. nested selectors
7. Mixin calls with @content

_Note to self: I likely need to go back through and deal with 7 -- I don't think I always put the media queries last._